docs/reference/docker-buildx-create.1

```diff
@@ -0,0 +1,325 @@
+.nh
+.TH "DOCKER-BUILDX-CREATE" "1" "Mar 2024" "" ""
+
+.SH NAME
+.PP
+docker-buildx-create - Create a new builder instance
+
+
+.SH SYNOPSIS
+.PP
+\fBdocker buildx create [OPTIONS] [CONTEXT|ENDPOINT]\fP
+
+
+.SH DESCRIPTION
+.PP
+Create makes a new builder instance pointing to a Docker context or endpoint,
+where context is the name of a context from \fBdocker context ls\fR and endpoint is
+the address for Docker socket (eg. \fBDOCKER_HOST\fR value).
+
+.PP
+By default, the current Docker configuration is used for determining the
+context/endpoint value.
+
+.PP
+Builder instances are isolated environments where builds can be invoked. All
+Docker contexts also get the default builder instance.
+
+
+.SH OPTIONS
+.PP
+\fB--append\fP[=false]
+	Append a node to builder instead of changing it
+
+.PP
+\fB--bootstrap\fP[=false]
+	Boot builder after creation
+
+.PP
+\fB--buildkitd-config\fP=""
+	BuildKit daemon config file
+
+.PP
+\fB--buildkitd-flags\fP=""
+	BuildKit daemon flags
+
+.PP
+\fB--driver\fP=""
+	Driver to use (available: "docker-container", "kubernetes", "remote")
+
+.PP
+\fB--driver-opt\fP=[]
+	Options for the driver
+
+.PP
+\fB-h\fP, \fB--help\fP[=false]
+	help for create
+
+.PP
+\fB--leave\fP[=false]
+	Remove a node from builder instead of changing it
+
+.PP
+\fB--name\fP=""
+	Builder instance name
+
+.PP
+\fB--node\fP=""
+	Create/modify node with given name
+
+.PP
+\fB--platform\fP=[]
+	Fixed platforms for current node
+
+.PP
+\fB--use\fP[=false]
+	Set the current builder instance
+
+
+.SH EXAMPLE
+.EX
+### Append a new node to an existing builder (--append) {#append}
```

Comment:
Looks like these first headings in the examples section don't get the .SS markup